src/database/DataTypes/Transaction.js

```diff
@@ -114,7 +114,9 @@ export class Transaction extends Realm.Object {
       this.otherParty.masterLists.forEach(masterList => {
         const itemsToAdd = complement(masterList.items, this.items, item => item.itemId);
         itemsToAdd.forEach(masterListItem => {
-          if (!masterListItem.item.crossReferenceItem) {
+          if (!masterListItem.item.crossReferenceItem
+            && database.objects('Item')
```

Comment:
masterListItem.item.isVisible

Reply:
I did not realize masterlist had visible field. Will do this.

Reply:
When you get a realm db object, you can access related records just like object